### PR TITLE
terminal-icons: Add version 0.9.0

### PR DESCRIPTION
**Pull request by sitiom**
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/terminal-icons.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "version": "0.9.0",
+    "description": "A PowerShell module to show file and folder icons in the terminal",
+    "homepage": "https://github.com/devblackops/Terminal-Icons",
+    "license": "MIT",
+    "notes": [
+        "Use the module by running: 'Import-Module Terminal-Icons'",
+        "Add it to your $PROFILE to make it permanent"
+    ],
```
Comment (niheaven) [inline bucket/terminal-icons.json:9]:
```suggestion
    "license": "MIT",
```

Don't add usage notes in manifest.

Comment (sitiom) [reply bucket/terminal-icons.json:9]: Its not a usage note, I just followed the other manifests in this repo ([`zlocation`](https://github.com/ScoopInstaller/Extras/blob/59b802f3e6b9b9b52c3c632d449afb48f54f023c/bucket/zlocation.json), [`powertab`](https://github.com/ScoopInstaller/Extras/blob/59b802f3e6b9b9b52c3c632d449afb48f54f023c/bucket/powertab.json), [`posh-docker`](https://github.com/ScoopInstaller/Extras/blob/59b802f3e6b9b9b52c3c632d449afb48f54f023c/bucket/posh-docker.json)).

Comment (niheaven) [reply bucket/terminal-icons.json:9]: In fact, it is, and the others' should be cleaned up too.

But okay, keep it now and batch clean job will be done later.

**Review by niheaven — APPROVED** [review]

**Comment by rashil2000** [comment]
Add a note "Nerd font is required for this module to work".

(suggest cannot be added because there can be multiple nerd-fonts).

**Pull request merged**